scene/resources/line_shape_2d.cpp

```diff
@@ -115,7 +115,7 @@ void LineShape2D::_bind_methods() {
 LineShape2D::LineShape2D() :
 		Shape2D(Physics2DServer::get_singleton()->line_shape_create()) {
 
-	normal = Vector2(0, -1);
+	normal = Vector2(0, 1);
```

Comment:
For the reference, this likely breaks compat. But I guess it's for the better?

Reply:
It should only affect new LineShape2D objects, and also, the line is the same as long as d = 0 since they are both horizontal lines at y = 0. It mostly just makes the default direction downwards.